client/src/shared/components/Icon/assets/edit.tsx

```
@@ -0,0 +1,27 @@
+import { SVGProps } from 'react';
+
+export const Edit = ({ width, height, color, ...props }: SVGProps<SVGSVGElement>) => (
+  <svg
+    width={width ?? 24}
+    height={height ?? 24}
+    viewBox="0 0 24 24"
+    fill="none"
+    xmlns="http://www.w3.org/2000/svg"
+    {...props}
+  >
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)